package.json

```diff
@@ -16,5 +16,6 @@
   "devDependencies": {
     "husky": "4.3.8",
     "lerna": "^4.0.0"
-  }
+  },
+  "dependencies": {}
```

Comment:
@AhmedIbrahim336 is this necessary?

Reply:
It happened because I installed a package in the wrong directory! I just removed it